### PR TITLE
Add Grai to non-free

### PR DESCRIPTION
**Pull request by ieaves**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [X] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [ ] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [X] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Miscellaneous`.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] If you are adding software forked from another project, please provide/link to a clear list of differences between both.
- [X] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: non-free.md [hunk 1]
```diff
@@ -155,10 +155,12 @@
 ### Software Development - Testing
 
 - [Bamboo](https://www.atlassian.com/software/bamboo) - Continuous integration server. `⊘ Proprietary` `Java`
+- [Grai](https://www.grai.io) `⊘ Proprietary` - Grai is the open source data observability tool that catches bugs before they make it to prod. Using data lineage it understands how changes impact your data stack and reports back in version control. ([Source Code](https://www.github.com/grai-io/grai-core)) `Elastic-2.0` `Python/Typescript`
```
Comment (KieranRobson) [inline non-free.md:158]: > https://github.com/grai-io/grai-core/tree/master/examples/deployment

```suggestion
- [Grai](https://www.grai.io) `⊘ Proprietary` - Grai is the open source data observability tool that catches bugs before they make it to prod. Using data lineage it understands how changes impact your data stack and reports back in version control. ([Source Code](https://www.github.com/grai-io/grai-core)) `Elastic-2.0` `Docker`
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
One quick change to be made.

File: non-free.md (hunk 1 above)
Comment (ieaves) [reply non-free.md]: Done! Thanks @KieranRobson

File: non-free.md [hunk 2]
```diff
@@ -218,6 +218,7 @@
 **[`^        back to top        ^`](##awesome-selfhosted---non-free-software)**
 
 - [Bamboo](https://www.atlassian.com/software/bamboo) - Continuous integration server. `⊘ Proprietary` `Java`
+- [Grai](https://www.grai.io) `⊘ Proprietary` - Grai is the open source data observability tool that catches bugs before they make it to prod. Using data lineage it understands how changes impact your data stack and reports back in version control. ([Source Code](https://www.github.com/grai-io/grai-core)) `Elastic-2.0` `Docker`
```
Comment (nodiscc) [inline non-free.md:221]: - The description should explain what the software does
- If the software is actually under the Elastic-2.0 license, please remove the `⊘ Proprietary` label.

Comment (ieaves) [reply non-free.md]: @nodiscc

I've attempted to reword the description more to your liking but please let me know if you need any additional modifications.

> If the software is actually under the Elastic-2.0 license, please remove the ⊘ Proprietary label.

Apologies, this followed from the description build utility. I had assumed it was a required label in the non-free.

**Pull request merged**